packages/elements/src/internals/machines/sign-up/router.types.ts

```diff
@@ -26,6 +26,7 @@ export const SignUpRouterSteps = {
   verification: 'step:verification',
   callback: 'step:callback',
   error: 'step:error',
+  restrictedAccess: 'step:restricted-access',
```

Comment:
why we call it `step:restricted-access` instead of `step:restricted`?

Reply:
@nikospapcom I just felt like appending `access` made it a bit clearer. Happy to change it, if you feel strongly about removing it.

Reply:
Not a strong opinion about that, we can proceed with what we have

Reply:
@nikospapcom Wound up changing it to your suggestion. 👍